plugin/src/main/java/com/denizenscript/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -85,6 +85,7 @@ public static void registerCommands() {
         registerCommand(ResetCommand.class);
         registerCommand(ZapCommand.class);
         // entity
+        registerCommand(SmoothEntityDataCommand.class); // <---------

```

Comment:
why the comment

Reply:
Will remove later, was a ~~badly made~~ TODO for myself